lib/core/fijkplayer.dart

```diff
@@ -429,11 +429,12 @@ class FijkPlayer extends ChangeNotifier implements ValueListenable<FijkValue> {
     } else if (!isPlayable()) {
       FijkLog.e("$this invoke seekTo invalid state:$state");
       return Future.error(StateError("Non playable state $state"));
-    } else {
-      FijkLog.i("$this invoke seekTo msec:$msec");
-      _seeking = true;
-      _channel.invokeMethod("seekTo", <String, dynamic>{"msec": msec});
     }
+
+    FijkLog.i("$this invoke seekTo msec:$msec");
+    _seeking = true;
+    _channel.invokeMethod("seekTo", <String, dynamic>{"msec": msec});
+    _currentPos = Duration(milliseconds: msec);
```

Comment:
Thanks for this PR.
seekTo is an async call to native player,   update `_currentPos` to seek `msec` here. It may be updated unexpectedly before native player actually make seek operation.

Reply:
Sure. Would you prefer I add `await` to the channel method invocation?

Reply:
I prefer this https://github.com/befovy/fijkplayer/blob/ef1243358f8e39b427ff4603c7a46d38df600693/lib/ui/panel2.dart#L337 
save seek time in your custom panel files.

Or change the native ijkplayer to update position after seek even player is paused.

Reply:
Why not just `await` the seek response?

Reply:
The seek operation in ijkplayer is still async, so `await seekTo` in fijkplayer make no sense.

You can make a new `Stream<>` wrapper , forward value from  `FijkPlayer.currentPo` to your new Stream , and handel seek case in it.